mlprof/tasks/parameters.py

```diff
@@ -12,6 +12,47 @@
 from mlprof.tasks.base import BaseTask
 
 
+class Model():
+
+    def __init__(self, model_file: str, name, label, **kwargs):
+
```

Comment:
Very strictly speaking not necessary, but we should still add a `super()` here.

Reply:
done